proto/storage/node.proto

```diff
@@ -77,8 +77,14 @@ message NodeScan {
     UNSUPPORTED = 1;
     KERNEL_UNSUPPORTED = 2;
     CERTIFIED_RHEL_CVES_UNAVAILABLE = 3;
+    UNSUPPORTED_NODE_INDEXER = 4;
```

Comment:
It feels that this change is isolated from the rest of this PR. Was this meant to be added here or in a follow-up?

Reply:
This served as a reminder in the interoperability discussion. I've removed this for now and will re-add it should we need it.
Thanks for flagging that to me